src/showcase/showcase-changes.less

```diff
@@ -179,11 +179,15 @@
     }
 }
 
+.multi.rsi.indicator {
+    stroke: @tertiary-purple;
+}
```

Comment:
Okay, for some reason it reset to D3FC's default, and `@tertiary-purple` was applied, but for some reason the order of precedence changed and the showcase CSS is being overriden by D3FC CSS.
Even though on Bitflux it works in the correct order of precedence.